src/ui/symbollayer/widget_hashline.ui

```diff
@@ -475,10 +475,10 @@
        </property>
       </widget>
      </item>
-     <item row="4" column="0" colspan="5">
+     <item row="0" column="0" colspan="5">
       <widget class="QCheckBox" name="mCheckPlaceOnEveryPart">
        <property name="toolTip">
-        <string>If checked, the &quot;first vertex&quot; or &quot;last vertex&quot; placements will rendered at the first and last vertex for every part of multipart geometries</string>
+        <string>If checked, the &quot;first vertex&quot; or &quot;last vertex&quot; placements will also render at the first and last vertex for every part of multipart geometries</string>
        </property>
        <property name="text">
         <string>Place on every part</string>
```

Comment:
How about?
```suggestion
        <string>Place on every part extremity</string>
```

Reply:
Does this option also affect the "central point" placement mode? Depending on whether it does then this refinement might not be accurate